### PR TITLE
Camel routing example

### PR DESCRIPTION
**Pull request by zharkovalexzander**

### CODE REVIEW

File: src/main/java/com/streamforge/realm/twitch/oauth/dtos/TwitchUserTokenDto.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package com.streamforge.realm.twitch.oauth.dtos;
+
+import java.util.List;
+
+public class TwitchUserTokenDto {
+    private String access_token;
```
Comment (SerhiiDenysenkoGit) [inline src/main/java/com/streamforge/realm/twitch/oauth/dtos/TwitchUserTokenDto.java]: Is it named not in CamelCase in order to make marshalling/unmarshalling easily?

Comment (zharkovalexzander) [reply src/main/java/com/streamforge/realm/twitch/oauth/dtos/TwitchUserTokenDto.java]: Yeah, that one only is the example, todo job is to re-map to normal naming and pass it to the rest client as entity.

Comment (zharkovalexzander) [reply src/main/java/com/streamforge/realm/twitch/oauth/dtos/TwitchUserTokenDto.java]: Could we export this DTO somehow to rest or vice versa - create it just on rest, send the payload to rest as it came from twitch and map it there?

File: src/main/java/com/streamforge/realm/twitch/oauth/camel/TwitchAuthRoute.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+    private final Consumer<Exchange> HTTP_QUERY_PARAMS_TO_MAP = exchange -> {
+        Message in = exchange.getIn();
+        String queryParams = (String) in.getHeader(Exchange.HTTP_QUERY);
+        List<NameValuePair> mappedParams = new ArrayList<>();
+        try {
+            URIBuilder uriBuilder = new URIBuilder(Q_MARK + queryParams);
+            mappedParams = uriBuilder.getQueryParams();
+        } catch (URISyntaxException e) {
+            e.printStackTrace();
```
Comment (SerhiiDenysenkoGit) [inline src/main/java/com/streamforge/realm/twitch/oauth/camel/TwitchAuthRoute.java]: Could you log this error instead of using prinStackTrace, please

Comment (zharkovalexzander) [reply src/main/java/com/streamforge/realm/twitch/oauth/camel/TwitchAuthRoute.java]: Sure, thats a lot of work to do

File: src/main/java/com/streamforge/realm/twitch/oauth/dtos/TwitchUserTokenDto.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    public Long getExpires_in() {
+        return expires_in;
+    }
+
+    public String getRefresh_token() {
+        return refresh_token;
+    }
+
+    public void setAccess_token(String access_token) {
```
Comment (SerhiiDenysenkoGit) [inline src/main/java/com/streamforge/realm/twitch/oauth/dtos/TwitchUserTokenDto.java]: Is it possible to return instance after setting value (return this) to simplify our code?

Comment (zharkovalexzander) [reply src/main/java/com/streamforge/realm/twitch/oauth/dtos/TwitchUserTokenDto.java]: Oh, there is a bottleneck here, that I really wish to discuss. Let`s talk about this one in private mode.

File: src/main/java/com/streamforge/realm/twitch/oauth/dtos/TwitchUserTokenDto.java (hunk 1 above)
Comment (SerhiiDenysenkoGit) [reply src/main/java/com/streamforge/realm/twitch/oauth/dtos/TwitchUserTokenDto.java]: I think it's applicable, will do it

**Review by SerhiiDenysenkoGit — APPROVED** [review]

**Pull request merged**